src/bitcoind/mod.rs

```diff
@@ -23,6 +23,11 @@ use jsonrpc::{
     simple_http,
 };
 
+/// Number of retry the client is allowed to do in case of timeout and i/o error
+/// while communicating with the bitcoin daemon.
+/// A retry happens every 1 second, after one minute the client stop trying.
```

Comment:
supernit
```suggestion
/// Number of retries in case of timeout or i/o error
/// while communicating with the bitcoin daemon.
/// A retry happens every 1 second, this makes us give up after one minute.
```